rca/project_styleguide/templates/patterns/atoms/profile-image/profile-image.html

```diff
@@ -0,0 +1,6 @@
+{% load wagtailimages_tags %}
+
+<div class="profile-image {% if modifier %}profile-picture--{{ modifier }}{% endif %}">
```

Comment:
should the modifier class be `profile-image--{{ modifier }}` rather than `profile-picture--`?

Reply:
oops, missed during refactor! Thanks!